Libraries/Linking/NativeLinking.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+'use strict';
+
+import type {TurboModule} from 'RCTExport';
+import * as TurboModuleRegistry from 'TurboModuleRegistry';
+import Platform from 'Platform';
+
+export interface Spec extends TurboModule {
+  // Common interface
+  +getInitialURL: () => Promise<string>;
+  +canOpenURL: (url: string) => Promise<boolean>;
+  +openURL: (url: string) => Promise<void>;
+  +openSettings: () => Promise<void>;
+
+  // Android only
+  +sendIntent: (
+    action: string,
+    extras: ?Array<{key: string, value: string | number | boolean}>,
+  ) => Promise<void>;
+
+  // RCTEventEmitter
```

Comment:
Could we update this to say `// Events`, RCTEventEmmitter is iOS specific and we're also dropping the RCT convention

Reply:
Yupp !

Reply:
cc @fkgozali, i think this came from your comment on the thread, just want to double check that it wasn't for a purpose i'm missing

Reply:
This is iOS specific and the exact super class name is RCTEventEmitter - so it's fine to leave it that way.